docs/EthStats/Lite-Network-Health-Monitor.md

```diff
@@ -0,0 +1,49 @@
+
+## Statistics Overview
+* List of nodes in the network with specific metrics regarding its last received block like number, hash, transaction and uncle count, block time and propagation time and other metrics like connected peers, if it's mining or not, hash rate, latency, uptime.
+* Charts for Block Time, Block Difficulty, Block Gas Limit, Block Uncles, Block Transactions, Block Gas Used, Block Propagation Histogram and Top miners
+* IP based geolocation overview
+* Node logs (displays the data sent by a node)
+* Block history (ability to go back in time and playback the block propagation throughout the nodes)
+
+## Github repositories
+* [ethstats-cli](https://github.com/Alethio/ethstats-cli) - client app that extracts data from a node and sends it to the server 
+* [ethstats-network-server](https://github.com/Alethio/ethstats-network-server) - service that consumes the data received from the nodes through the client app 
+* [ethstats-network-dashboard](https://github.com/Alethio/ethstats-network-dashboard) - front end dashboard  
+
+## Installation
+
+The installation process is done through multiple steps. First the server needs to be installed with its dependencies and then for every node in the network the client app `ethstats-cli` is needed.
+
+#### Server
+
+In the server's repo there are 2 examples of `docker-compose` files to set it up in lite mode with:
+* [Memory persistence](https://github.com/Alethio/ethstats-network-server/blob/master/docker/lite-mode/memory-persistence/docker-compose.yml) - the consumed data will be persisted in memory in a json data model and in case of a crash/restart the data is lost
+* [Redis persistence](https://github.com/Alethio/ethstats-network-server/blob/master/docker/lite-mode/redis-persistence/docker-compose.yml) - the consumed data will be persisted into Redis 
+
+The examples contains all the needed dependencies like: 
+* [Deepstream](https://github.com/deepstreamIO/deepstream.io) - for sending event based data to the dashboard in real time
+* [ethstats-network-dashboard](https://github.com/Alethio/ethstats-network-dashboard) - front end dashboard
+* [Redis](https://redis.io/) - if persistence is needed
+
+More details about installing and running are available in the github [repository](https://github.com/Alethio/ethstats-network-server).
+#### EthStats-CLI
+
+After the server is up and running the next step is to install `ethstats-cli` for the nodes in the network. This can be done through: 
+* [NPM](https://github.com/Alethio/ethstats-cli#install) - the package needs to be installed globally and also can be used as a [daemon](https://github.com/Alethio/ethstats-cli#daemon)
+* [Docker](https://github.com/Alethio/ethstats-cli#docker) - images available on [docker hub](https://hub.docker.com/r/alethio/ethstats-cli)
+
+More details about installing and running are available in the github [repository](https://github.com/Alethio/ethstats-cli).  
```

Comment:
It'd be really nice to see an end to end example (either using Docker or installing) of setting these things up with Pantheon.  Do you think that's something could be included?

Reply:
Sure thing, added examples.